### DISCUSSION TITLE
not able to appear image automatically in device resolution.

### DISCUSSION THREAD

**Discussion by ghost** [axmolengine/axmol#633] [Q&A] [answered]
hello devs,
hope you all are doing well..  i am new in adxe and i have started recently. now issue is that actually when i was run project on iphone 8 it's perfectly able to run but i have run it on ipad or any other simulator it's always appearing the size of iphone 8 or may be size of first time ran device. and one blue line is always appear in all simulator.. kindly please do let me know the solution.. i am attaching the screenshot for your reference.. thanks in advance all of you for your time.

![Screenshot 2022-05-13 at 5 32 08 PM](https://user-images.githubusercontent.com/105281914/168280584-b581fd1c-c59d-42db-859b-da1a5036b517.png)
.

**Comment by DelinWorks** [comment, marked as answer]
hi there, for the resolution i really don't know, but for the blue lines thats actually the screen's safe area being displayed, you will find it as a DrawNode object in the scene file, just remove it and the blue lines will disappear.

**Reply by ghost** [reply]
thanks @DelinWorks brother for reply yes, blue line is disappeared.. thanks but still issue is there about resolution. if you have any idea about how i can set a code for all multiple resolution device adaption in adxe..

**Reply by DelinWorks** [reply]
Look something up in google about cocos2dx multi resolutions, if that doesn't solve it, look for something like, design resolution.

**Reply by ghost** [reply]
okay brother, i already searched but i did not found more convenient.. but i am trying thanks @DelinWorks

**Reply by DelinWorks** [reply]
Sorry for not helping much I literally have 0 experience with apple development.

**Reply by ghost** [reply]
okay brother understood, but i am looking for it in android and in cocos2dx resolution policy is same for both ios and android as i understood, only question is that how i could manage same object(sprites, ui and all) position in all multiple device resolution(device visible area)..  again thanks.

**Reply by DelinWorks** [reply]
Yes, that's design scale, you choose a fixed resolution for your game and it will look the same on any other device, and resolution policy for how you want to treat the design scale resolution. hope that helps.

**Reply by DelinWorks** [reply]
I don't know about how cocos2dx treats it's ui, but you can simply create a ui system of your own that aligns to edges, follow screen sizes and what not.

**Reply by ghost** [reply]
okay brother i am trying hardly thank for your kind support

**Reply by Abo7mood** [reply]
@DelinWorks wanna contact you, my email is Nalmuqbil@odel.sa
